wiki/osu!_Program_Files/en.md

```diff
@@ -14,13 +15,10 @@ needs_cleanup: true
 | Current default filepath | Old installer filepath |
 | :-- | :-- |
 | `C:\Users\<Username>\AppData\Local\osu!` | `C:\Program Files\osu!` / `C:\Program Files(x86)\osu!` (for 7 64-bit or 8) |
-> `C:` is your drive with osu! installed.
 
-## Folders
-
-### \_MacOS
+> `C:` is your drive with osu! installed.
 
-**osu! for Macintosh OS.** For Windows OS users, just ignore it or delete it if you wish.
+<!--## Folders-->
```

Comment:
that's still a proper section heading, please uncomment it

Reply:
Fixed.